src/Css.elm

```diff
@@ -4317,66 +4382,1913 @@ listStyle3 (Value val1) (Value val2) (Value val3) =
 
 
 
--- BORDER STYLE --
+{- BORDERS -}
+
 
+{-| Sets [`border`](https://css-tricks.com/almanac/properties/b/border/) property.
+
+    border (px 1)
+    border2 (px 1) solid
+    border3 (px 1) solid (hex "#f00")
 
-{-| A `wavy` [text decoration style](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-style#Values).
 -}
-wavy : Value { provides | wavy : Supported }
-wavy =
-    Value "wavy"
```

Comment:
I've removed `wavy` value, because it's related with `text-decoration-style` which hasn't implemented yet and I thought it can confuse a contributor who will implement it. Should we keep changes or come `wavy` back?

Reply:
I think it's fine to leave it. It already happens that implementors run into things that have already been implemented (e.g. `left` is reused in several places), so I think this should be fine!